src/app/routes/index.test.js

```diff
@@ -1,9 +1,29 @@
 import routes from './index';
-import { cpsAssetPagePath } from './regex';
+import {
+  cpsAssetPagePath,
+  legacyAssetPagePath,
+  articlePath,
+  radioAndTvPath,
+  frontPagePath,
+  errorPagePath,
+} from './regex';
+import {
+  MEDIA_ASSET_PAGE,
+  STORY_PAGE,
+  PHOTO_GALLERY_PAGE,
+  FEATURE_INDEX_PAGE,
+  INDEX_PAGE,
+  ARTICLE_PAGE,
+  LIVE_RADIO_PAGE,
+} from './pageTypes';
 
+jest.mock('#pages/Article', () => jest.fn());
```

Comment:
could just be `jest.mock('#pages/Article', jest.fn);`

Reply:
I tried to implement this, but couldn't get it to work - error message was:
`the second argument of jest.mock must be a function`

I will leave it as is for now, if that's OK.